### DISCUSSION TITLE
Is it possible to restrict summary to one line?

### DISCUSSION THREAD

**Discussion by BlueManCZ** [dunst-project/dunst#1390] [Q&A]
Hello, 
is there a way to configure Dunst so that the notification summary stays on a single line? Currently, when the summary is long, it wraps over multiple lines and, with specific height set, takes up the entire notification space, resulting in this appearance:
```
notify-send "Lorem ipsum notification lorem ipsum notification lorem ipsum notification" "This notification is for testing purposes"
```
![obrazek](https://github.com/user-attachments/assets/4df309ed-451b-465d-9d7d-da538679660e)

I'm trying to achieve something similar to this:
![obrazek](https://github.com/user-attachments/assets/67bac6eb-65c4-49e4-90e0-be927861d967)

Even just simply cut out without ellipsis would be fine.

I went through the [`dunst(5)`](https://man.archlinux.org/man/dunst.5.en) and [Pango markdown](https://docs.gtk.org/Pango/pango_markup.html) but didn't find anything useful or similar to HTML's `nowrap` attribute or CSS's `white-space`.

**Comment by bynect** [comment]
Hello, I think that currently the ellipsize property is applied to the whole layout, so to do what you want we would need to modify and provide an extra ellipsize property.
I like this idea, so we may add it in the future
